pkg/api/norman/store/cluster/cluster_store.go

```diff
@@ -136,6 +139,12 @@ func GetClusterStore(schema *types.Schema, mgmt *config.ScaledContext, clusterMa
 		NodeLister:                    mgmt.Management.Nodes("").Controller().Lister(),
 		DialerFactory:                 mgmt.Dialer,
 	}
+
+	dynamicClient, err := dynamic.NewForConfig(&mgmt.RESTConfig)
```

Comment:
I don't think we could hit an error here since other places will fatal on a bad mgmt.RESTConfig but we should at least log the error at a warning level so we have a record in the logs that we don't have the dynamic client

Reply:
Updated logging a warning if fails